src/dashboard/src/templates/archival_storage/list.html

```diff
@@ -96,7 +96,11 @@
       <tbody>
         {% for item in aips %}
           <tr>
-            <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }}</a></td>
+              {% if item.encrypted %}
+              <td><a href="{% url 'view_aip' item.uuid %}">{{ item.name }} ({% trans "encrypted" %})</a></td>
```

Comment:
Ok. I see you removed the parens too. Did you mean to do that? To me, they indicate that the "encrypted" string is a qualifier on the name of the AIP and not the part of the name itself, i.e., "My AIP (encrypted)" vs "My Other AIP". Are you fine with::

```
    {% blocktrans with aip_name=item.name %}{{ aip_name }} (encrypted){% endblocktrans %}
```

and then the translator can remove the parens if they see fit?